resources/css/styles.css

```diff
@@ -202,6 +202,10 @@
  *                                                                         *
  **************************************************************************/
 
+#validationImgView {
+
+}
```

Comment:
You did not specify anything in here... so if there is no styling planned (as I assume so), you can delete this here.